com.unity.cinemachine/Runtime/Upgrader/CinemachineUpgrader.cs

```diff
@@ -0,0 +1,577 @@
+#if UNITY_EDITOR
+// #define DEBUG_HELPERS
+// suppress obsolete warnings
+#pragma warning disable CS0618 
+
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEditor.SceneManagement;
+using UnityEngine;
+using UnityEngine.Playables;
+using UnityEngine.SceneManagement;
+using UnityEngine.Timeline;
+
+namespace Cinemachine.Upgrader
```

Comment:
Why is this in Runtime and not in Editor?

Reply:
Should be in Editor.